oop_hw3_v2/train.py

```diff
@@ -0,0 +1,20 @@
+from random import randint
+from faker import Faker
+
+fake = Faker()
+
+
+class Train:
```

Comment:
When using len on a train, I want to see a list of wagons(trainCars) without a locomotive.